docs/modules/tiered-storage/pages/overview.adoc

```diff
@@ -7,22 +7,26 @@
 
 {description}
 
-NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage. See xref:deploy:enterprise-licenses.adoc[].
+NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage.
+See xref:deploy:enterprise-licenses.adoc[].
 
 == What Can You Do with Tiered Storage?
 
 Have fewer Hazelcast members for large maps. For example, for a map with 1 TB of data, you'll need
 ten Hazelcast members if all data is kept in memory and each member has 100 GB of data. Tiered Storage allows you to split the data between memory and disks, which requires fewer cluster members depending on how the data is distributed.
 
+== When to Use Tiered Storage
 
-== How Does Tiered Storage Work?
+Accessing data in the memory offers superior access speed, but memory is much more expensive than disks, while the modern NVMe SSDs offer performance that makes Tiered Storage a viable alternative to memory-only storage at a much lower total cost of ownership.
 
-Tiered Storage uses the following:
+Consider using Tiered Storage if one of the following is true for your use-case.
```

Comment:
An additional use case to consider:
You have significant temporal access patterns inside a larger dataset

Reply:
Added in 06a2a152059bfb69e1201dfdbf7842c9b65f589f

Reply:
Minor update: 
You have significant temporal access patterns inside a very large dataset.

I would also move this up to be positioned below the bullet 'the hot set of entries...'